src/backend/verilog.rs

```diff
@@ -21,6 +21,15 @@ use vast::v17::ast as v;
 #[derive(Default)]
 pub struct VerilogBackend;
 
+// based on type of mem, determines what string you should use to access mem
```

Comment:
This function needs better and more exact documentation on what happens for which memories